djarg/tests/views.py

```diff
@@ -95,6 +95,12 @@ class GrantStaffObjectView(
     success_url = '.'
 
 
+class GrantStaffIfSameNameObjectView(GrantStaffObjectView):
+    queryset = arg.func(
+        lambda request: User.objects.filter(first_name=request.user.first_name)
```

Comment:
I needed some/any function that would accept `request` as an argument, and use it in some way to filter down the queryset. I figured, it'd be simple (and kinda silly) to filter to the same name as the requesting user.